### PR TITLE
As an end-user, I should be able to see the list of agents in the widget.

### LINKED ISSUE #437: As an end-user, I should be able to see the list of agents in the widget.
- [x] API to list the available agents
- [ ] Update UI to show the list of agents.

### PR DESCRIPTION
**Pull request by nithindavid**
Fixes #437 

**Screenshot**
Screenshot 2020-02-03 at 1 26 47 PM

### CODE REVIEW

File: app/javascript/widget/components/GroupedAvatars.vue [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<template>
+  <div class="avatars">
+    <span v-for="user in users" :key="user.id" class="avatar">
+      <Thumbnail
+        size="24px"
+        :username="user.name"
+        status="online"
+        :src="user.avatar"
```
Comment (pranavrajs) [inline app/javascript/widget/components/GroupedAvatars.vue]: `user.avatar_url`

File: app/javascript/widget/store/modules/agent.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import Vue from 'vue';
+import { getAvailableAgents } from 'widget/api/agent';
+
+const state = {
+  agents: [],
+  isError: false,
```
Comment (pranavrajs) [inline app/javascript/widget/store/modules/agent.js]: I would prefer to have it nested under `uiFlags`

File: app/javascript/widget/components/AvailableAgents.vue [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+  },
+  computed: {
+    users() {
+      return this.agents.map(agent => ({
+        id: agent.id,
+        url: agent.avatar_url,
+        name: agent.name,
+      }));
+    },
+    title() {
```
Comment (pranavrajs) [inline app/javascript/widget/components/AvailableAgents.vue]: Could you move this outside as a helper and cover it with a test case.

File: app/javascript/widget/components/AvailableAgents.vue [hunk 4]
```diff
@@ -0,0 +1,90 @@
+<template>
+  <div class="avaiable-agents">
```
Comment (pranavrajs) [inline app/javascript/widget/components/AvailableAgents.vue]: `available` spelling mistake

**Comment by CL545740896** [comment]
Maybe  the list of agents  can  show  when i first open the widget .  Can it show it every times when i open the widget ?

**Comment by pranavrajs** [comment]
@CL545740896 That's a good suggestion, we will see how we can do it.

**Pull request merged**